packages/grid/_modules_/grid/hooks/features/rows/useGridParamsApi.ts

```diff
@@ -78,9 +78,17 @@ export function useGridParamsApi(apiRef: GridApiRef) {
     (id: GridRowId, field: string) => {
       const colDef = apiRef.current.getColumn(field);
       const value = apiRef.current.getCellValue(id, field);
-      const baseParams = getBaseCellParams(id, field);
+      const row = apiRef.current.getRow(id);
       const params: GridCellParams = {
-        ...baseParams,
```

Comment:
I got an improvement from 20ms to 8ms by removing this spread operator.

Reply:
Wow, it's pretty wild 👏!

On the full 100,000 rows data set, filtering goes from

<img width="449" alt="Capture d’écran 2021-06-24 à 18 57 06" src="https://user-images.githubusercontent.com/3165635/123303419-1308a180-d51e-11eb-9ed3-a1990d34ab36.png">

https://material-ui.com/components/data-grid/#commercial-version

to

<img width="460" alt="Capture d’écran 2021-06-24 à 18 57 12" src="https://user-images.githubusercontent.com/3165635/123303423-13a13800-d51e-11eb-8a64-2545b5084d4b.png">

https://deploy-preview-1929--material-ui-x.netlify.app/components/data-grid/#commercial-version